revision/views.py

```diff
@@ -0,0 +1,26 @@
+from reversion.models import Version
+from reversion.views import RevisionMixin
+from reversion.errors import RegistrationError
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from rest_framework.exceptions import APIException
+from .serializers import RevisionSerializer
+
+
+class CustomRevisionMixin(RevisionMixin):
+
+    def get_versions(self):
+        instance = self.get_object()
+        try:
+            versions = Version.objects.get_for_object(instance)
+        except RegistrationError:
+            raise APIException(detail='model has not been registered for revision.')
+        current_version = versions[0]
+        versions = versions.exclude(pk=current_version.id)
+        return instance, versions
+
+    @action(methods=['GET'], detail=True)
+    def instance_logs(self, request, **kwargs):
+        instance, versions = self.get_versions()
+        serializer = RevisionSerializer(versions, many=True)
+        return Response(serializer.data)
```

Comment:
If the number of versions for an instance can be large, consider adding pagination to the `instance_logs` response to improve performance and usability.

Reply:
At the moment in project we don't have this logic anywhere and it might be complicated. Can we skip it for now? If there will be time i will add it.